src/lib/spinel/radio_spinel.hpp

```diff
@@ -727,14 +727,16 @@ template <typename InterfaceType, typename ProcessContextType> class RadioSpinel
     /**
      * This method checks whether the spinel interface is radio-only.
      *
-     * @param[out] aSupportsRcpApiVersion   A reference to a boolean variable to update whether the list of spinel
-     *                                      capabilities include `SPINEL_CAP_RCP_API_VERSION`.
+     * @param[out] aSupportsRcpApiVersion          A reference to a boolean variable to update whether the list of
+     *                                             spinel capabilities include `SPINEL_CAP_RCP_API_VERSION`.
+     * @param[out] aSupportsMinHostRcpApiVersion   A reference to a boolean variable to update whether the list of
+     *                                             spinel capabilities include `SPINEL_CAP_RCP_MIN_HOST_API_VERSION`.
      *
      * @retval  TRUE    The radio chip is in radio-only mode.
      * @retval  FALSE   Otherwise.
      *
      */
-    bool IsRcp(bool &aSupportsRcpApiVersion);
+    bool IsRcp(bool &aSupportsRcpApiVersion, bool &aSupportsMinHostRcpApiVersion);
```

Comment:
nit: to be consistent with `SPINEL_CAP_RCP_MIN_HOST_API_VERSION`.
```suggestion
    bool IsRcp(bool &aSupportsRcpApiVersion, bool &aSupportsRcpMinHostApiVersion);
```